webapp/channels/src/utils/constants.tsx

```diff
@@ -1391,6 +1391,16 @@ export const DefaultRolePermissions = {
     ],
 };
 
+export const ModeratedPermissions = [
```

Comment:
1/5 I think it would be useful to leave a comment describing why we need this constant